api/bin/chainflip-cli/src/settings.rs

```diff
@@ -88,6 +88,16 @@ pub enum BrokerSubcommands {
 	/// Request a swap deposit address.
 	RequestSwapDepositAddress(SwapRequestParams),
 	WithdrawFees(WithdrawFeesParams),
+	/// Register this account as a broker account.
+	RegisterAccount,
+	/// De-register this broker account.
+	DeregisterAccount {
+		#[clap(
+			help = "Force deregistration. If the account has any funds, they will be lost.",
+			long = "force"
```

Comment:
we should add a default / make the default clear to the user (and the default should be false ofc)

Reply:
I thought the default was implicit, will check again.

Reply:
Yeah I think it is for bools, but we should include this in the message then. Doesn't show the default in the message

Reply:
Behaviour is that if the `chainflip-cli broker deregister-account` runs with force-false and `chainflip-cli broker deregister-account --force` runs it with true.